modules/va_facilities/lib/tasks/facilities_tasks.rake

```diff
@@ -7,6 +7,6 @@ namespace :va_facilities do
       UPDATE base_facilities
       SET location=ST_GeogFromText('SRID=4326;POINT(' || long || ' ' || lat ||')')
     SQL
-    ActiveRecord::Base.conection.execute sql
+    ActiveRecord::Base.connection.execute sql
```

Comment:
interesting.. I wonder if this is why the locations are blank for cemeteries?

Reply:
🤦‍♂️

Reply:
It's actually most NCA, VBA, and VC facilities that have blank locations. No idea how _some_ of them aren't blank, or how all the VHA facilities are ok.